extension/js/common/platform/util.ts

```diff
@@ -32,3 +34,8 @@ export const moveElementInArray = <T>(arr: Array<T>, oldIndex: number, newIndex:
   arr.splice(newIndex, 0, arr.splice(oldIndex, 1)[0]);
   return arr;
 };
+
+export const iso2022jpToUtf = (content: Buf) => {
+  const decoder = new TextDecoder();
+  return decoder.decode(content);
```

Comment:
Is it guessing the encoding? I think it would be better to tell it which encoding to decode.

Reply:
Yes, it's guessing now,
I will take a look at how it will behave
Thanks!